Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.yml

```diff
@@ -2131,9 +2131,90 @@ script:
     description: Updates the remote incident with local incident changes. This method is only used for debugging purposes and will not update the current incident.
     execution: false
     name: update-remote-system
+  - arguments:
+      - description: Access key name.
+        name: name
+        required: true
+      - description: Timestamp in milliseconds when access key expires. Default:0.
+        name: expires-on
+    deprecated: false
+    description: Adds a new access key for the current user. If you have API access, you can create up to two access keys.
```

Comment:
Is it correct for all kinds of users? Also for admins? if yes ignore this comment and resolve the conversation.